src/pykx/pandas_api/pandas_indexing.py

```diff
@@ -453,6 +453,29 @@ def rename(self, labels=None, index=None, columns=None, axis=0,
                 t = _rename_columns(t, columns)
 
         return t
+    
+    def add_suffix(self, suffix, axis=0):
+        t = self
+        if suffix:
+            if axis == 0:
+                raise ValueError('nyi')
+            elif axis == 1:
+                c_str = 'cols value' if "Keyed" in str(type(t)) else 'cols'
+                return q(f'{{(c!`$string[c:{c_str} y],\:string x)xcol y}}', suffix, t)
+            else:
+                raise ValueError(f'No axis named {axis}')
+
+    def add_prefix(self, prefix, axis=0):
+        t = self
+        if prefix:
+            if axis == 0:
+                raise ValueError('nyi')
+            elif axis == 1:
+                c_str = 'cols value' if "Keyed" in str(type(t)) else 'cols'
+                return q(f'{{(c!`$string[x],/:string c:{c_str} y)xcol y}}', prefix, t)
+            else:
+                raise ValueError(f'No axis named {axis}')
+        return t
```

Comment:
Unreachable code, either assign `t` to the inner expression or remove this line. Keep it consistent among both implementations.